core/src/main/java/edu/wpi/grip/core/operations/composite/FindBlobsOperation.java

```diff
@@ -66,6 +74,8 @@ public FindBlobsOperation(InputSocket.Factory inputSocketFactory, OutputSocket.F
         inputSocket,
         minAreaSocket,
         circularitySocket,
+        convexitySocket,
+        inertiaRatioSocket,
```

Comment:
Unfortunately, if you don't put these last, you'll break deserializing older save files.

Reply:
Ah gotcha, thanks for the heads up for the future.